lib/cc/services/github_pull_requests.rb

```diff
@@ -39,8 +39,8 @@ def report_status?
   end
 
   def should_apply_rollout?
-    (github_login.present? || github_user_id.present?) &&
-      (config.rollout_usernames.present? || config.rollout_percentage.present?)
+    (github_login.present? || config.rollout_usernames.present?) &&
+      (github_user_id.present? || config.rollout_percentage.present?)
```

Comment:
I think the conditional operators need to change here as well.

(github_login.present? && config.rollout_usernames.present?) ||
  (github_user_id.present? && config.rollout_percentage.present?)